src/js/component.js

```diff
@@ -815,21 +815,21 @@ class Component {
   /**
    * Add a CSS class name to the `Component`s element.
    *
-   * @param {string} classToAdd
-   *        CSS class name to add
+   * @param {string[]} classesToRemove
```

Comment:
I think `string[]` denotes an array of strings, but the code suggests that it takes any number of arguments and each is a string. I'm not sure that `string[]` is the right notation here. Without looking it up, I think it may be `string...`.

Reply:
Close! Looks like it's `{...string}` https://jsdoc.app/tags-param.html#multiple-types-and-repeatable-parameters

Reply:
Thx, I have just changed that.